docs/docs/gatsby-style-guide.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Gatsby Style Guide
+---
+
+The Gatsby community is building a massive, searchable "Gatsby Guides." This reference tool will eventually include thousands of articles that cover all topics relevant to people building on Gatsby - all written to be easily understood by the awesome community of people who love building with Gatsby!
+
+This repo is where we plan, write, and maintain these Guide articles, which we then host at Gatsby Guides.
```

Comment:
This feels a bit unclear — the guides will be hosted on gatsbyjs.org not on a separate domain. At least for now. This could be reconsidered in the future but in general I like keeping things together.

Reply:
Perhaps we split the documentation between API/code documentation and "guides" which is a catch-all for everything else.